example/example.ino~

```diff
@@ -0,0 +1,27 @@
+#include <sorter.h>
```

Comment:
Please delete this file. Backups (files ending with `~`) should not be part of git repository.